activerecord/lib/active_record/railties/databases.rake

```diff
@@ -2,8 +2,7 @@ require 'active_record'
 
 db_namespace = namespace :db do
   task :load_config do
-    ActiveRecord::Base.configurations       = ActiveRecord::Tasks::DatabaseTasks.database_configuration || {}
-    ActiveRecord::Migrator.migrations_paths = ActiveRecord::Tasks::DatabaseTasks.migrations_paths
```

Comment:
you're right, this is still needed. This should be address in another change we  the 'source of truth' is clearly defined: use `ActiveRecord::Tasks::DatabaseTasks.migrations_paths` instead `ActiveRecord::Migrator.migrations_paths` everywhere...